ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/dorhook.js

```diff
@@ -33,7 +33,7 @@
             return {
                 src : options.getPath() + '?resourceType=' + encodeURIComponent(DOR_DIALOG_PATH),
                 isFloating : false,
-                loadingMode : "sidePanel",
+                loadingMode : "auto",
```

Comment:
did we analyse what are the ripple effects of ?

Reply:
Had a discussion with @nit23uec a while back on this. This is actually necessary for any dialog in core components, since nothing should not (and is not supposed to) load on the left side panel (like in AF v1).
@devgurjar requesting your comments on this.

Reply:
Reverting this as this breaks Dor EditConfig.